keps/sig-node/3857-rro-mounts/README.md

```diff
@@ -631,9 +638,23 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - run RecursiveReadOnly="Enabled", and verify that the mount is actually recursively read-only
 - run RecursiveReadOnly="Disabled", and verify that the mount is actually not recursively read-only
 
-Tests are implemented in <https://github.com/kubernetes/kubernetes/blob/v1.30.0/test/e2e_node/mount_rro_linux_test.go>,
-and will be executed on the CI when the CI is upgraded to use containerd v2.0.
-So, there is no link to the testgrid yet.
+The `e2e_node` tests are implemented in <https://github.com/kubernetes/kubernetes/blob/v1.30.0/test/e2e_node/mount_rro_linux_test.go>.
```

Comment:
Commenting here, but this applies to other parts of the doc:
1. In `##### Unit tests` I'm missing explicit coverage and a list of affected packages, just like we required in [the template](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#unit-tests)

Reply:
Added